src/pages/post/index.tsx

```diff
@@ -60,33 +71,50 @@ const PostPage = (): ReactElement => {
   const handleUpdateCategory: SelectOnChangeHandler = ({ code }) => {
     setPostForm({
       ...postForm,
-      category: Number(code)
+      category: code
     })
   }
 
-  // const handleUpdateImageUrls: UploaderOnChangeHandler = ({ images }): void => {
-  //   setPostForm({
-  //     ...postForm,
-  //     imageUrls: images
-  //   })
-  // }
+  const handleUpdateImageInfos: UploaderOnChangeHandler = async ({
+    images: imageInfos
+  }) => {
+    setPostForm(prev => ({
+      ...prev,
+      imageInfos
+    }))
+  }
 
   const handleUpdatePostForm: HandleUpdatePostForm = (e): void => {
-    const { name, value } = e.currentTarget
+    const { name, value } = e.target
 
     setPostForm({
       ...postForm,
       [name]: value
     })
   }
 
-  const handleChangeRadio = (e: ChangeEvent<HTMLFormElement>) => {
-    const { name, value } = e.target
+  const handlePostProduct = async () => {
+    const { imageInfos, price, ...restInfos } = postForm
+    const imageFormData = new FormData()
 
-    setPostForm({
-      ...postForm,
-      [name]: Number(value)
+    imageInfos.forEach(info => {
+      if (info.file) {
+        imageFormData.append('files', info.file)
+      }
+    })
```

Comment:
```suggestion
imageInfos
  .filter(info => info.file)
  .forEach(info => imageFormData.append('files', info.file))
```
선언형으로 바꾸면 더 가독성이 높아질 것 같습니다!

Reply:
필터로는 file 유무에 대한 타입 추론이 안되어서 forEach만 사용하도록 변경해봤습니다 ! 

[변경완료](https://github.com/price-offer/offer-fe/pull/179/commits/17abbec909f742c99514d4e2c6f59e8da13e77e8) !

Reply:
좋습니다!

> 필터로는 file 유무에 대한 타입 추론이 안되어서

혹시 요 부분은 어떤 부분인가요??

Reply:
filter로 file 유무를 걸러내도 forEach에서 append시 file이 없을 수도 있다는 에러가 발생해요 !